miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/js/MiniAppMessageBridge.kt

```diff
@@ -111,6 +116,12 @@ abstract class MiniAppMessageBridge {
         this.isAdMobEnabled = isAdMobProvided()
     }
 
+    /** Set implemented userInfoHandler. Can use the default provided class from sdk [UserInfoHandler]. **/
+    fun setUserInfoHandler(handler: UserInfoHandler) {
+        this.userInfoHandler = handler
+        this.userInfoHandler.init(this, customPermissionCache, miniAppInfo.id)
```

Comment:
Unfortunately, this does not work as we expected.

The `customPermissionCache` and `miniAppInfo` were initialized only when webview is created. There is a case that hostapp developer can set `userInfoHandler` before webview is created, `MiniAppMessageBridge` object is separated from miniapp webview.

@corycaywood I think we should go back with the solution init `userInfoHandler` in `MiniAppMessageBridge` constructor.

Reply:
It's okay to pass the only the `MiniAppMessageBridge` instead of passing those parameters separately. That's how, we don't need to worry about when miniapp will be created. 

```
fun setUserInfoHandler(handler: UserInfoHandler) {
        this.userInfoHandler = handler
        this.userInfoHandler.init(miniAppBridge)
    }
```

Reply:
but it will cause cyclical dependency because `MiniAppMessageBridge` contains `UserInfoHandler`.

Maybe there is a solution to keep using `setUserInfoHandler` and still allow developers to set it before webview is created.